Files/Converters/BooleanToVisibilityConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+
+namespace Files.Converters
+{
+    class BooleanToVisibilityConverter : IValueConverter
```

Comment:
You can use WCT built-in [converter](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.converters.booltovisibilityconverter?view=win-comm-toolkit-dotnet-stable&viewFallbackFrom=uwp-toolkit-dotnet).

Reply:
I didn't realize there was one. Thanks!